gcc/config/riscv/corev.md

```diff
@@ -103,13 +107,14 @@
 
 (define_insn "cv_mulhhsn"
   [(set (match_operand:SI 0 "register_operand" "=r")
-    (lshiftrt:SI
-      (mult:SI (sign_extend:SI (lshiftrt:SI
-			         (match_operand:SI 1 "register_operand" "r")
-				 (const_int 16)))
-	       (sign_extend:SI (lshiftrt:SI
-			         (match_operand:SI 2 "register_operand" "r")
-				 (const_int 16)))
+    (ashiftrt:SI
+      (mult:SI 
+        (sign_extend:SI 
+          (lshiftrt:SI (match_operand:SI 1 "register_operand" "r")
+				               (const_int 16)))
+	      (sign_extend:SI 
+          (lshiftrt:SI (match_operand:SI 2 "register_operand" "r")
+				               (const_int 16)))
```

Comment:
The indentation looks funny here

Reply:
Noted. I'll go through the indentation again and make sure it looks right.